src/amo/components/AddonsByAuthorsCard/index.js

```diff
@@ -105,61 +107,81 @@ export class AddonsByAuthorsCardBase extends React.Component<Props> {
       numberOfAddons,
       showSummary,
       type,
+      showMore,
     } = this.props;
 
     if (!loading && (!addons || !addons.length)) {
       return null;
     }
 
     let header;
+    const headerModifier = showMore ? 'More' : '';
     switch (addonType) {
       case ADDON_TYPE_DICT:
         header = i18n.ngettext(
           i18n.sprintf(
-            i18n.gettext('More dictionaries by %(author)s'),
-            { author: authorDisplayName }
+            i18n.gettext('%(headerModifier)s dictionaries by %(author)s'),
```

Comment:
Unfortunately we cannot do it this way, because we cannot assume anything about where the `headerModifier` will be placed in other languages, plus we need the headerModifier to be able to be translated into other languages.

So we need to wrap the full string, in both cases, in `i18n.gettext`. This makes for a bit more code, but I don't think there's anything we can do to avoid it. You basically need to wrap both the `Dictionaries by %(author)s` and the `More dictionaries by %(author)s` separately in `i18n.gettext`.

Reply:
hmm...ok ...i'll try again : )